_assets/css/_fullwidth.scss

```diff
@@ -1066,15 +1030,15 @@ Stylesheet: Full Width Page Stylesheet
 // Events
 // ------
 
-#events {
+div#events {
```

Comment:
This is to address the possibility that there is a heading called "Events", since Kramdown puts an id on all headings that are added in Markdown. I had an issue with another id like that. Actually, looking at this some more, we should be able to turn off that option, which might be the best thing to do.

Reply:
Auto IDs turned off, so this is resolved.